operator/helm/helm.go

```diff
@@ -152,36 +155,62 @@ func (c *Client) Install(name string, chartInfo ChartInfo, opts InstallOpts) err
 	client.Timeout = opts.Timeout
 	client.Wait = opts.Wait
 
-	if _, err := client.Run(chart, opts.Values); err != nil {
-		return fmt.Errorf("failed to install Helm chart %q: %w", name, err)
+	rel, err := client.Run(chart, opts.Values)
+	if err != nil {
+		return nil, fmt.Errorf("failed to install Helm chart %q: %w", name, err)
 	}
 
-	return nil
+	return rel, nil
 }
 
-// Uninstall satisfies the Uninstaller interface. It uninstalls a Helm
-// installation using its name.
-func (c *Client) Uninstall(name string, opts UninstallOpts) error {
-	// TODO: implement.
+// Uninstall uninstalls a Helm release by name. Uninstall returns nil if the release doesn't exist.
+func (c *client) Uninstall(name string, opts UninstallOpts) error {
+	namespace := opts.Namespace.ValueOrDefault()
+	cfg, err := c.config(namespace)
+	if err != nil {
+		return fmt.Errorf("failed to uninstall Helm release %q: %w", name, err)
+	}
+	client := action.NewUninstall(cfg)
+	client.Description = opts.Description
+	if _, err := client.Run(name); err != nil {
+		if errors.Is(err, driver.ErrReleaseNotFound) {
+			return nil
+		}
+		return fmt.Errorf("failed to uninstall Helm release %q: %w", name, err)
+	}
 	return nil
 }
 
-// Get satisfies the Get interface. It gets a Helm installation using its name.
-func (c *Client) Get(name string, opts GetOpts) (*Release, error) {
+// Get gets a Helm installation using its name.
+func (c *client) Get(name string, opts GetOpts) (*release.Release, error) {
 	namespace := opts.Namespace.ValueOrDefault()
 	cfg, err := c.config(namespace)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get Helm release %q: %w", name, err)
 	}
 	client := action.NewGet(cfg)
-	res, err := client.Run(name)
+	rel, err := client.Run(name)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get Helm release %q: %w", name, err)
 	}
-	return &Release{res}, nil
+	return rel, nil
+}
+
+// ListResources lists the Kubernetes resources for a given Helm release.
```

Comment:
Should we say something about the resources here not necessarily matching the server state? (I believe you just return what helm renders, not the resources that are actually deployed.)

Reply:
Good idea to have a better description. Fixed with eb01c31.